web/src/containers/StateDashboard.js

```diff
@@ -0,0 +1,101 @@
+import PropType from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Icon, { faPlusCircle, faSpinner } from '../components/Icons';
+import Sidebar from './StateDashboardSidebar';
+import TopBtns from './TopBtns';
+import { createApd, selectApd } from '../actions/apd';
+import { Section } from '../components/Section';
+import { t } from '../i18n';
+
+const Loading = () => (
+  <div className="h2 p0 pb3 center">
+    <Icon icon={faSpinner} spin size="sm" className="mr1" /> Loading APDs
+  </div>
+);
+
+const StateDashboard = ({
+  apds,
+  createApd: create,
+  fetching,
+  selectApd: select,
+  state
+}) => {
+  const open = id => e => {
+    e.preventDefault();
+    select(id);
+  };
+
+  return (
+    <div className="site-body">
+      <Sidebar />
+      <div className="site-main p2 sm-p4 md-px0">
+        <TopBtns hideDashboard />
+
+        <Section resource="stateDashboard">
+          <div className="mb3 bg-white rounded shadow accordian">
+            <div className="px3 py2 border-bottom border-bottom-darken-1 blue">
+              <button
+                className="btn bg-blue white rounded p1 right"
+                size="small"
+                onClick={create}
+              >
+                Create new&nbsp;&nbsp;
+                <Icon icon={faPlusCircle} />
+              </button>
+              <span className="h2">{state.name} APDs</span>
+            </div>
+            <div className="p3">
+              {fetching ? <Loading /> : null}
+              {!fetching && apds.length === 0 ? t('stateDashboard.none') : null}
+              {apds.map(apd => (
+                <div key={apd.id} className="p2 mb2 bg-gray-lightest">
+                  <div className="inline-block p2 mr2 bg-white blue">
+                    <img
+                      src="/static/img/icon-document.svg"
+                      alt=""
+                      className="align-middle"
+                    />
+                  </div>
+                  <a href="#!" className="h3 bold" onClick={open(apd.id)}>
```

Comment:
And an `h3`

Reply:
Hm, could we make it an `h3` element that wraps an `a` tag?